scripts/test.sh

```
@@ -4,39 +4,69 @@ TEMP_DIR=$(mktemp -d)
 trap 'rm -rf "$TEMP_DIR"' EXIT
 
 go mod tidy
-cd health
+
 # Run health tests with coverage
+cd health
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)